catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/LibraryImpl.java

```diff
@@ -143,6 +135,10 @@ public void setOutgoingValidationEnabled(boolean outgoingValidationEnabled) {
         this.outgoingValidationEnabled = outgoingValidationEnabled;
     }
 
+    public void setEmailSender(EmailSender emailSender) {
```

Comment:
I would add javadoc indicating that emailSender shouldn't be null and add a 'notNull' check in the code.